underlying-test-case.md

```diff
@@ -79,6 +79,14 @@ use Illuminate\Foundation\Testing\RefreshDatabase;
 uses(TestCase::class, RefreshDatabase::class)->in(__DIR__);
 ```
 
+In some cases it is not enough to just add the trait with `uses()` you can also call a function on the trait. 
+Can be done like this: 
+```php
+uses(CreatesApplication::class);
+beforeEach()
+    ->createApplication();
+```
```

Comment:
I'd probably put the trait method call onto the same line just to be a bit cleaner. Perhaps we could also change the wording to be more consistent with the other examples?

```suggestion
In some cases it is not enough to just add the trait with `uses()`, for these situations you can also call a function on the trait like so:

```php
uses(CreatesApplication::class);

beforeEach()->createApplication();
```
```